### ISSUE TITLE
Is it possible to change the ID to a class, so we can use multiple comment boxes on 1 page?

### ISSUE THREAD

**Issue by filtration** [laravelista/comments#5]
I have a timeline, where a user can comment on multiple statuses, but at the moment, this uses an ID, so will only render to 1, is it possible to change to a class so it renders to all on the page? I tried changing some of the code, but kept getting: 

Target container is not a DOM element

I changed:

    <div id="laravelista-comments"></div>

to 

    <div class="laravelista-comments"></div>

and in the app.js to:

```
import React from 'react';
import ReactDOM from 'react-dom';
import CommentBox from './components/CommentBox.jsx';

window.Laravelista = window.Laravelista || {content_type: null, content_id: null, login_path: '/login'};

var elementLength = document.getElementsByClassName("laravelista-comments").length;
if (elementLength > 0) {
    for (var i = 0; i < elementLength; i++) {
        ReactDOM.render(
            <CommentBox content_type={Laravelista.content_type} content_id={Laravelista.content_id} login_path={Laravelista.login_path} />,
            document.getElementsByClassName('laravelista-comments')[i] //mountNode
        );
    }
}
```

Is there anyway to achieve this?

**Comment by mabasic** [comment]
Currently no, but I will find a way to get this working when I get to updating this package.

**Comment by mabasic** [comment]
Fixed in version [2.0.0](https://github.com/laravelista/comments/releases/tag/2.0.0)
